src/bucket.js

```diff
@@ -71,6 +81,7 @@ export default class Bucket {
    * @param  {Object} options.headers The headers object option.
    * @return {Promise<Object, Error>}
    */
+  @nobatch("Batch doesn't support read operations.")
```

Comment:
I don't think the client should prevent something that the server accept as valid.